.github/ISSUE_TEMPLATE.md

```diff
@@ -1,20 +1,64 @@
-###### What version of Jekyll are you using (`jekyll -v`)?
+<!--
+  Hi! Thanks for considering to file a bug with Jekyll.  Please take the
+  the basic questions (convert `[ ]` into `[x]` to check it (or submit and check.)
+  If are no need for certain fields like output and redirection, please delete
+  those headers before submitting.  We know not all tickets require those
+  steps.  Otherwise, please try to be as detailed as possible.
+  
+  If you are unsure this is a bug in Jekyll, or this is a bug caused
+  by a plugin that isn't directly related to Jekyll, or if this is just
+  a generic usage question, please consider asking your question at
+  https://talk.jekyllrb.com where non-bug questions go.
+  
+  Thanks!
+-->
 
+- [ ] I believe this to be a bug, not a question about using Jekyll.
+- [ ] I Read the CONTRIBUTION file at https://github.com/jekyll/jekyll/blob/master/.github/CONTRIBUTING.markdown
```

Comment:
This should be `I read the contributing documentation at https://jekyllrb.com/docs/contributing/`, right?